kurtosis-module/impl/ethereum_genesis_generator/genesis_generator.go

```diff
@@ -0,0 +1,218 @@
+package ethereum_genesis_generator
+
+import (
+	"github.com/kurtosis-tech/kurtosis-core-api-lib/api/golang/lib/enclaves"
+	"github.com/kurtosis-tech/kurtosis-core-api-lib/api/golang/lib/services"
+	"github.com/kurtosis-tech/stacktrace"
+	"github.com/sirupsen/logrus"
+	"io"
+	"os"
+	"path"
+	"strings"
+	"text/template"
+)
+
+const (
+	imageName                    = "kurtosistech/ethereum-genesis-generator"
+	serviceId services.ServiceID = "eth-genesis-generator"
+
+	// The filepaths, relative to shared dir root, where we're going to put EL & CL config
+	// (and then copy them into the expected locations on image start)
+	elGenesisConfigYmlRelFilepathInSharedDir = "el-genesis-config.yml"
+	clGenesisConfigYmlRelFilepathInSharedDir = "cl-genesis-config.yml"
+	clMnemonicsYmlRelFilepathInSharedDir = "cl-mnemonics.yml"
+
+	// The genesis generation image is configured by dropping files into a specific hardcoded location
+	// These are those locations
+	// See https://github.com/skylenet/ethereum-genesis-generator
+	expectedConfigDirpathOnService     = "/config"
+	expectedELConfigDirpathOnService   = expectedConfigDirpathOnService + "/el"
+	expectedELGenesisConfigYmlFilepath = expectedELConfigDirpathOnService + "/genesis-config.yaml"
+	expectedCLConfigDirpathOnService   = expectedConfigDirpathOnService + "/cl"
+	expectedCLGenesisConfigYmlFilepath = expectedCLConfigDirpathOnService + "/config.yaml"
+	expectedCLMnemonicsYmlFilepath = expectedCLConfigDirpathOnService + "/mnemonics.yaml"
+
+	// The generator container hardcodes the location where it'll write the output to; this is the location
+	outputGenesisDataDirpathOnGeneratorContainer = "/data"
+
+	// Location, relative to the root of shared dir, where the genesis data will be copied to from the generator container
+	//  so that the module container can store and use it
+	genesisDataRelDirpathInSharedDir = "data"
```

Comment:
is this missing the "/" in the path?

Reply:
oo that's actually an interesting question - it's definitely a dirpath (this could have been `data/foobar/bang`), relative to the shared dir root, but we haven't actually standardized on whether relative paths should begin with `/`. I'd so far been taking the course of "no" in our code & Bash scripts since the `/` will get added when you join the parent with the relative path, but I don't have a strong opinion about it.

Reply:
ok, now I understand why, thanks